bin/node/testing/benches/import.rs

```diff
@@ -92,10 +92,169 @@ fn genesis(keyring: &BenchKeyring) -> node_runtime::GenesisConfig {
 //     //endowed-user//01
 //      ...
 //     //endowed-user//N
+#[derive(Clone)]
 struct BenchKeyring {
 	accounts: BTreeMap<AccountId, sr25519::Pair>,
 }
 
+// This is prepared database with genesis and keyring
+// that can be cloned and then used for any benchmarking.
+struct BenchDb {
+	random_space: usize,
+	keyring: BenchKeyring,
+	directory_guard: Guard,
+}
+
+impl Clone for BenchDb {
+	fn clone(&self) -> Self {
+		let random_space = self.random_space;
+		let keyring = self.keyring.clone();
+		let dir = tempdir::TempDir::new("sub-bench").expect("temp dir creation failed");
+
+		let seed_dir = self.directory_guard.0.path();
+
+		log::trace!(
+			target: "bench-logistics",
+			"Copying seed db from {} to {}",
+			seed_dir.to_string_lossy(),
+			dir.path().to_string_lossy(),
+		);
+		let seed_db_files = std::fs::read_dir(seed_dir)
```

Comment:
So copying files is faster than recreating them? How do you make sure that rocksdb has flushed all data to disc at this point?

Reply:
RocksDB is already dropped at this point, so should be flushed.

Copying seems faster than opening, letting genesis to instantiate everything and closing

Reply:
If RocksDB is dropped, it is good!